staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go

```diff
@@ -276,3 +276,10 @@ func (s byPrefixPriority) Less(i, j int) bool {
 
 	return strings.Compare(s[i], s[j]) < 0
 }
+
+// RedirectTo redirects request to a certain destination.
+func RedirectTo(to string) func(http.ResponseWriter, *http.Request) {
```

Comment:
this does not look like something that belongs into `mux`. Better turn it into a local helper in staging/src/k8s.io/apiserver/pkg/server/routes

Reply:
fixed.